backend/apps/residents/tests.py

```
@@ -0,0 +1,359 @@
+from django.test import RequestFactory
+from django_tenants.test.cases import FastTenantTestCase
+from django_tenants.test.client import TenantClient
+
+from apps.bedrooms.models import Bedroom
+from apps.common.services import build_access_token
+from apps.membership.models import Membership, Role
+from apps.residences.models import Residence, ResidenceDomain
+from apps.residents.permissions import IsResidenceAdmin
+
+UserModel = get_user_model()
+PASSWORD = "demo1234"  # NOSONAR
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "PASSWORD"

(S105)